packages/components/src/spinner/style.scss

```diff
@@ -0,0 +1,44 @@
+.components-spinner--svg {
```

Comment:
The SVG is working fairly well here! However just as far as classnames go, should we call it "track" instead? Also, I think it should be __ instad of --. I.e. `.components-spinner__track` for example.